web2/src/pages/settings/PlexSettingsPage.tsx

```diff
@@ -219,22 +259,221 @@ export default function PlexSettingsPage() {
             </FormControl>
           </Grid>
         </Grid>
-        <Box>
-          <FormControl></FormControl>
-          <FormControl>
-            <TextField defaultValue={streamSettings.audioCodecs} />
-          </FormControl>
-          <FormControl></FormControl>
-        </Box>
-      </Paper>
+      </>
+    );
+  };
+
+  const renderAudioSettings = () => {
+    if (streamSettingsPending) {
+      return (
+        <Paper sx={{ display: 'flex' }}>
+          <Box flex={1}>
+            <Skeleton width="70%">
+              <TextField fullWidth />
+            </Skeleton>
+          </Box>
+          <Box flex={1}>
+            <Skeleton width="70%">
+              <TextField fullWidth />
+            </Skeleton>
+          </Box>
+        </Paper>
+      );
+    }
+
+    if (!streamSettings) {
+      return <div>Error</div>;
+    }
+
+    return (
+      <>
+        <Typography component="h6" sx={{ my: 2 }}>
+          Audio Options
+        </Typography>
+        <Grid flex="1 0 50%" container spacing={3}>
+          <Grid item xs={12} sm={6}>
+            <FormControl fullWidth>
+              <TextField
+                id="component-outlined"
+                label="Audio Codecs"
+                defaultValue={streamSettings?.audioCodecs}
+              />
+            </FormControl>
+            {streamSettings.audioCodecs.map((codec) => (
+              <Chip
+                label={codec}
+                key={codec}
+                onDelete={(codec) => removeAudioCodec(codec)}
+                sx={{ mr: 1, mt: 1 }}
+              />
+            ))}
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <FormControl fullWidth>
+              <InputLabel id="maximum-audio-channels-label">
+                Maxium Audio Channels
+              </InputLabel>
+              <Select
+                labelId="maximum-audio-channels-label"
+                id="maximum-audio-channels"
+                label="Maxium Audio Channels"
+                value={streamSettings.maxAudioChannels}
+              >
+                <MenuItem value={1}>1.0</MenuItem>
+                <MenuItem value={2}>2.0</MenuItem>
+                <MenuItem value={2.1}>2.1</MenuItem>
```

Comment:
I think we should treat these as strings (theoretically there could be something like 3.1.2 in the future w/ height channels, who knows)